sktime/forecasting/hf_transformers_forecaster.py

```diff
@@ -40,7 +43,8 @@ class HFTransformersForecaster(BaseForecaster):
         Path to the huggingface model to use for forecasting. Currently,
         Informer, Autoformer, and TimeSeriesTransformer are supported.
     fit_strategy : str, default="minimal"
-        Strategy to use for fitting the model. Can be "minimal" or "full"
```

Comment:
I think it needs to be much clearer to the user how the fine-tuning is done here - or that `fit`, in fact, is fine-tuning.

Reply:
I think it should also be possible to fine-tune the model with time series that are later not used in the forecast - but that might require the global forecasting interface to be in place.

Reply:
That will not be an issue because once the model has been configured to use peft in train, it will remain the same in predict as well